external/vcm/vcm/catalog.yaml

```diff
@@ -998,3 +998,27 @@ sources:
         access: read_only
       urlpath: "gs://vcm-ml-code-testing-data/c8-grids-regression-testing/landseamask.zarr/"
       consolidated: true
+
+  grid/ne30:
+    description: Lat, lon, and area of NE30 data
+    driver: zarr
+    metadata:
+      grid: ne30
+      variables:
+        - area
+        - lat
+        - lon
+    args:
+      urlpath: "gs://vcm-scream/latLonArea/ne30/ne30.zarr/"
```

Comment:
I wonder if we should save this `area` data in units of `m^2` instead of the current `rad^2`? High potential for bugs, although I understand a desire to be consistent with SCREAM output.

Reply:
Also another question: did you reshape this data to 6 x 90 x 40? Or is that the native output shape?